scripts/quests/jeuno/LB05_1_Shattering_Stars.lua

```diff
@@ -0,0 +1,147 @@
+-----------------------------------
+-- Shattering Stars
+-----------------------------------
+-- Log ID: 3, Quest ID: 132
+-- Maat : !pos 8 3 118 243
+-----------------------------------
+require("scripts/settings/main")
+require("scripts/globals/items")
+require("scripts/globals/keyitems")
+require("scripts/globals/npc_util")
+require("scripts/globals/quests")
+require("scripts/globals/titles")
+require('scripts/globals/interaction/quest')
+local ID = require("scripts/zones/RuLude_Gardens/IDs")
+-----------------------------------
```

Comment:
ID var and formatting, also, could you make the last require quotes consistent with the others?  Switching from ' to "